doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -2609,3 +2609,24 @@ detect thread. For each output script, a single state is used. Keep in
 mind that a rule reload temporary doubles the states requirement.
 
 .. _deprecation policy: https://suricata.io/about/deprecation-policy/
+
+.. _suricata-yaml-config-hardening:
+
+Configuration hardening
+-----------------------
+
+The `security` section of suricata.yaml is meant to provide in-depth security configuration options.
+
+Besides landlock, (see :ref:`landlock`), one setting is available.
+`limit-noproc` is a boolean to prevent process creation by Suricata.
+If you do not need Suricata to create other processes (for LUA scripts for instance), enable this to
```

Comment:
and threads... A plugin for example may use threads - especially if it uses some Rust runtime like tokio where threads may be dynamically created as needed at runtime.

Reply:
indeed